src/test/kotlin/fandor/ReachEngineEventHandlerFunctionTest.kt

```diff
@@ -1,26 +1,54 @@
 package fandor
 
+import fandor.ReachEngineEventHandlerFunction
+import com.amazonaws.services.s3.event.S3EventNotification
+import com.fasterxml.jackson.annotation.JsonProperty
 import io.micronaut.context.ApplicationContext
 import io.micronaut.http.client.HttpClient
 import io.micronaut.runtime.server.EmbeddedServer
 import junit.framework.Assert.assertEquals
 import org.jetbrains.spek.api.Spek
 import org.jetbrains.spek.api.dsl.describe
 import org.jetbrains.spek.api.dsl.it
+import java.io.ByteArrayInputStream
 
 object ReachEngineEventHandlerFunctionTest: Spek({
     describe("BookController Suite") {
         var embeddedServer : EmbeddedServer = ApplicationContext.run(EmbeddedServer::class.java)
         var client : HttpClient = HttpClient.create(embeddedServer.url)
 
         it("test /hello responds Hello World") {
-            var rsp : String = client.toBlocking().retrieve("/hello")
-            assertEquals(rsp, "Hello World")
+            val entity:S3EventNotification.RequestParametersEntity = S3EventNotification.RequestParametersEntity( "70.90.214.57" )
```

Comment:
Please use a space to separate the variable name and colon from the type:
```suggestion
            val entity: S3EventNotification.RequestParametersEntity = S3EventNotification.RequestParametersEntity( "70.90.214.57" )
```